hms-api/src/service/hackathon-service.ts

```diff
@@ -35,8 +36,17 @@ export async function createHackathon(
     throw new ValidationError(`Cannot create Hackathon`, result);
   }
 
+  const category = new Category('General', 'General', hackathon.id);
+  const categoryResult = category.validate();
+  if (categoryResult.hasFailed()) {
+    throw new ValidationError(`Cannot create general Category`, categoryResult);
+  }
```

Comment:
you should also store this category in the DB I would imagine 🤔

Reply:
ah I should have kept reading, I see you do :D

Reply:
hmmm now that I think about it a bit more, it might be best to encapsulate all of this within the category-service